common/supplemental/supplementalData.xml

```diff
@@ -4787,91 +4787,91 @@ XXX Code for transations where no currency is involved
         <calendar type="islamic">
           <calendarSystem type="lunar" />
           <eras>
```

Comment:
there is some evidence for a BH era: https://web.archive.org/web/20170202005514/http://www.sharjah.ae/registration-locations.aspx?Val=111#:~:text=A%20limited%20number%20of%20years%20before%20Hijra%20(BH)%20are%20used%20to%20date%20events%20related%20to%20Islam%2C%20such%20as%20the%20birth%20of%20Prophet%20Mohammad%20in%2053%20BH.

Reply:
Yes I definitely agree that we should consider adding a `bh` era. https://github.com/unicode-org/icu4x/issues/5068#issuecomment-2741915472

I consider _changing_ the eras out-of-scope for the current PR, but I'm happy to scope-creep if we have consensus and it won't block the PR.

Reply:
https://unicode-org.atlassian.net/browse/CLDR-18464